games-strategy/freeciv/freeciv-2.5.10.recipe

```diff
@@ -0,0 +1,125 @@
+PATCHES="freeciv-$portVersion.patchset"
+ADDITIONAL_FILES="freeciv.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ -z "${ARCHITECTURES##*!$targetArchitecture*}" ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+GLOBAL_WRITABLE_FILES="
+	settings/freeciv/database.lua keep-old
+	"
+
+PROVIDES="
+	freeciv = $portVersion
+	app:FreeCiv = $portVersion
+	cmd:freeciv_manual$commandSuffix
+	cmd:freeciv_sdl$commandSuffix
+	cmd:freeciv_server$commandSuffix
+	lib:libfreeciv$secondaryArchSuffix
+	lib:libfreeciv_srv$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:liblua$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libsdl_1.2$secondaryArchSuffix
+	lib:libsdl_gfx$secondaryArchSuffix
+	lib:libsdl_image_1.2$secondaryArchSuffix
+	lib:libsdl_mixer_1.2$secondaryArchSuffix
+	lib:libsdl_ttf_2.0$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:liblua$secondaryArchSuffix >= 5.2
+	devel:liblzma$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_gfx$secondaryArchSuffix
+	devel:libsdl_image$secondaryArchSuffix
+	devel:libsdl_mixer$secondaryArchSuffix
+	devel:libsdl_ttf$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	autoreconf -vfi
+	CFLAGS="-O2 -pipe" runConfigure --omit-dirs binDir ./configure \
+		--bindir=$commandBinDir --enable-client=sdl --enable-fcmp=no \
+		--enable-sys-lua --enable-shared
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	mkdir -p $appsDir
+	ln -s $commandBinDir/freeciv-sdl $appsDir/Freeciv
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local INTERNAL="0"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@INTERNAL@|$INTERNAL|" \
+		$portDir/additional-files/freeciv.rdef.in > freeciv.rdef
+
+	addResourcesToBinaries freeciv.rdef \
+		$appsDir/Freeciv
+
+	addAppDeskbarSymlink $appsDir/Freeciv
+	mv $prefix/share/* $dataRootDir
+	rmdir $prefix/share
+	rm -f $libDir/*.a $libDir/*.la
```

Comment:
It would be nice to use `--disable-static` instead of `rm -f $libDir/*.a`

Reply:
This doesn't work: 
Warning: POLICY ERROR: development library entry "/packaging/freeciv_x86/lib/x86/libfreeciv-srv.la" should be placed in "develop/lib/x86"

Reply:
I think the reason why you are getting this policy error is because you did not keep the `rm -f $libDir/*.la`.

(BTW, I should have been more explicit in my previous comment, and make it clear that we still need to drop the libtool files.)

Reply:
Why use --disable-static then?

Reply:
To speed up (a little bit) the compilation, since we don't need the static lib.
If I understand correctly, when you used `--disable-static` you still got the `$libDir/libfreeciv-srv.la`, but the policy only allows such files to be in the `$developDir` tree.

If I'm not mistaken, using both `--disable-static` and `rm -f $libDir/*.la` will fix the policy error.

Reply:
Need to check, could be a leftover from previous build ... getting back on this :)
PS, I saw that we could add support for magickwand (imagemagick), would it be wise to include this?

Reply:
Wasn't a leftover :) but included MagickWand into the build :)